mne/viz/_3d.py

```diff
@@ -1898,6 +1898,7 @@ def _plot_stc(stc, subject, surface, hemi, colormap, time_label,
     if backend in ['pyvista', 'notebook']:
         kwargs["show"] = False
         kwargs["view_layout"] = view_layout
+        kwargs["silhouette"] = True
```

Comment:
This looks like it defaults to `True`, but really it should default to `False` and probably be exposed in `SourceEstimate.plot` and related functions. However I start to worry abit about parameter explosion in `SourceEstimate.plot`, maybe we need a `brain_kwargs : dict | None` argument that allows for passing these sorts of things. Eventually `ssao` might be in there for example.

Reply:
Similarly to `add_data_kwargs` then